Update the dynamic root custom query step example (#6977)
umbraco/UmbracoDocs: 15/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/content-picker.md

}
```

> Finally, you need to register the custom query step client side, and provide some further description.
> 
> This can be done in an `umbraco-package.json` file like the following:

Reviewer: ```suggestion
Finally, register the custom query step on the client side and provide a brief description.

You can do this in an `umbraco-package.json` file, as shown below:
```

```json
{
  "$schema": "../../umbraco-package-schema.json",
  "name": "My.Test.Extension",
  "version": "0.1.0",
  "extensions": [
    {
      "type": "dynamicRootQueryStep",
      "alias": "Umb.DynamicRootQueryStep.MyCustomStep",
      "name": "Dynamic Root Query Step: My Custom Step",
      "meta": {
        "queryStepAlias": "MyCustomStep",
        "label": "My Custom Step",
        "description": "My custom step description.",
        "icon": "icon-coffee"
      },

>       "weight": 60

Reviewer 2: Let's leave out the `weight`? or set it to `0`

Then they can pick one if they are interested in positioning their option?

the core ones are between 60–100, so 0 would be in the bottom.

Author: Have set to zero.  I imagine that's the default if not provided, but it's probably useful to show that this is how you adjust the ordering.
